models/src/main/scala/coop/rchain/models/IsForgeable.scala

```diff
@@ -0,0 +1,6 @@
+package coop.rchain.models
+
+trait IsForgeable[T] {
```

Comment:
Please add a unit test exploring a couple of cases:

- one for a positive result (is fograble)
- one for a negative result (is fograble)
- a negative and a positive result involving nesting.

Reply:
Done.